mypy.ini

```diff
@@ -7,7 +7,7 @@ warn_unused_ignores = True
 [mypy - lib]
 ignore_errors = True
 
-[mypy-chia.data_layer.*,tests.core.data_layer.*,chia.cmds.data,chia.cmds.data_funcs,chia.rpc.data_layer_rpc_api,chia.rpc.data_layer_rpc_client,chia.server.start_data_layer,chia.wallet.db_wallet.*,chia.wallet.dlo_wallet.*,chia.wallet.util.merkle_tree,tests.wallet.db_wallet.*]
+[mypy-chia.cmds.data,chia.cmds.data_funcs,chia.data_layer.*,chia.rpc.data_layer_rpc_api,chia.rpc.data_layer_rpc_client,chia.server.start_data_layer,chia.wallet.db_wallet.*,chia.wallet.dlo_wallet.*,chia.wallet.util.merkle_tree,tests.core.data_layer.*,tests.wallet.db_wallet.*,tests.wallet.rpc.test_dl_wallet_rpc]
```

Comment:
A reorder plus adding `tests.wallet.rpc.test_dl_wallet_rpc` to the stricter list.  Note that recently on `main` strict has become the default so this list will get deleted when that is merged into atari.